src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: all;
```

Comment:
@bradrich This is also a strange edge case that I don't think is intuitive to people using `propagateContainerEvents`. They'd have to look at the source code to figure it out. The user shouldn't have to care about how we implemented the propagation, it should just work.